### ISSUE TITLE
Program menu icons became ~50% larger

### ISSUE THREAD

**Issue by bloojoop** [smeighan/xLights#3249]
NOTE: IF YOU DO NOT ATTACH A SEQUENCE THAT DEMONSTRATES YOUR PROBLEM THEN THERE IS A HIGH PROBABILITY YOUR ISSUE WILL JUST BE CLOSED AND NOT INVESTIGATED.

**Describe the bug**
I upgraded from 2022.09 to 2022.10 and the primary icons (new sequence,open seqence, save, etc) all became about 50% bigger.  The entire bar no longer fits on the screen. 

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)
upgrade to 2022.10. 

**Expected behavior**
expect icons to stay same size as previous versions 

**Screenshots**
attached a pic of the comparison.  No changes other than upgrade.  downgrade returns icons to original size.

**Versions (please complete the following information):**
xlights 2022.10. 
Windows 10 
Microsoft surface 3 laptop (3:2 aspect ratio)
screen resoulution manufactures recommended 2256x1504 with 150% scale and layout (I hate scale but this is what is highly recommended for this hardware). 

**Additional context**
not sequence related but attached a packaged sequence anyway. 

**Attachments**
- If associated with a crash please attach/link to a dropbox containing the crash log that was produced
- If it crashed but no crash log was produced please use Tools Menu/Package Logs and attach/link to a dropbox containing the resulting zip file
- If the error relates to rendering an effect or a crash while sequencing please use Tools Menu/Package Sequence and attach/link to a dropbox containing the resulting zip file

If you would prefer not to post your sequences publically please say so and be ready to PM them to one of the developers when asked.
<img width="696" alt="xlights icons" src="https://user-images.githubusercontent.com/88390168/168871759-582c32e9-219f-4973-b665-5f72835f9bac.png">
[largeicon.zip](https://github.com/smeighan/xLights/files/8710320/largeicon.zip)

**Comment by derwin12** [comment]
Are you sure you havent some how changed your scale and layout from the default 100% (Right click on desktop, display settings)
![image](https://user-images.githubusercontent.com/4643499/168927590-2b68b778-6fbd-4722-a354-8a46138905b7.png)

**Comment by derwin12** [comment]
Dev just posted on fb that xlights is now honoring the setting above .. so if you have it set to > 100%, xlights will now be using that value. Prior versions it did not.

**Comment by bloojoop** [comment]
yes this system is set to 150% which is the manufacturers recommended setting.
<img width="392" alt="scale" src="https://user-images.githubusercontent.com/88390168/168932452-008daa8a-881b-491a-a541-3349460cd507.png">

if you look at the images above, the icons are greatly out of proportion to all others.  More and more new systems are coming this this scaling set to >100% (personaly I don't like it).especially when coming with 2k or 4k displays.  Should we be telling users to scale back and lower their resolution away from manufacturers recommendations?  If so I'm OK with that just curious.

**Comment by computergeek1507** [comment]
It may be before xLights was manually applying the scaling and now the newer wx is also applying the scaling as well. The new wx also added icon scaling which could be causing it as well.

**Comment by dkulp** [comment]
The newer wxWidgets does a lot more automatic adjustment of UI elements based on the system settings.   That likely does break various places where we manually had to do it in the past.     One improvement I'd love to leverage at some point is that all of these icons COULD be SVG's that would scale nicely (instead of pixelated).   That said, I don't believe we have SVG's for any of the icons.   I really wish we would have collected the SVG's from the originals when we updated all the icons.   :(

**Comment by kppankratz** [comment]
Dan - I've got the original Illustrator files that I could easily save as SVG when needed.

**Comment by AzGilrock** [comment]
Yeah my FB comment was meant to be a joke with the possibility of being true but most likely due to wxWidgets

**Comment by derwin12** [comment]
btw as far as I know the smaller row of icons (the effects) are just controlled by the preferences.

**Comment by dkulp** [comment]
I believe this is now fixed, or at least improved.  It's going to require all the devs to update their wxWidgets builds and such so it may take some time to get into builds.

**Comment by derwin12** [comment]
This continues to show up in the zoom room. Usual cause is big screens with high res monitors/tvs and they have their display properties set to 300%. If the reduce that then cant really see the windows icons etc.

**Comment by jholmes09** [comment]
I came to GitHub to report the same issue but discovered you already know.
In 2022.13, I am noticing the sizing is highly inconsistent among icons & buttons.
I am on a 4K monitor with 300% scaling (recommended).

For example, the Edit menu is huge but every other menu in the top bar is 'normal.'

Meanwhile, on the Layout tab, the buttons for each model type are noticably smaller than the effects bar.

I am also discovering the windows themselves seem to default in all sorts of weird ways.
The screen-shot shows the default look when I begin a new sequence. See how the windows near the top are all smushed?

![image](https://user-images.githubusercontent.com/18503894/177465394-90615e5d-abd3-4cd2-9603-b95368e2ee33.png)

**Comment by chadgms** [comment]
I'm seeing the same thing.  I have three monitors, two of them are set to 100% the third(main) is set to 175% (I can't see anything on it at 100% with my old eyes :))  If I set the main to 100% it works fine.   If I set just that one, the tool bar and visual icons show like below.   Even if xLights launches UI on one of those from start (splash screen always starts on main monitor though).

Version 2022.14 64 bit
wxWidgets 3.1.7

At 175% setting
![175%scale](https://user-images.githubusercontent.com/47008482/181756781-c2043d3d-f11e-4dae-8dc7-794eecc28a07.jpg)

At 100% setting
![100percentscale](https://user-images.githubusercontent.com/47008482/181756817-83ad9a59-fe6e-4964-8195-e7a3c3ae8f7e.jpg)
